media/js/firefox/accounts.js

```diff
@@ -20,6 +22,11 @@
         fxaIframeSrc = fxaIframeSrc.replace('context=iframe', 'context=fx_firstrun_v2');
     }
 
+    // if email address provided in querystring, send to FxA frame
+    if (params.get('fxa-email')) {
+        fxaIframeSrc += '&email=' + params.get('fxa-email');
```

Comment:
@shane-tomlinson - I just took a guess at this, but it appears to work locally and on [demo5](https://www-demo5.allizom.org/en-US/firefox/sync/). Is this official, or as yet unfinished/in development?